testing/images/clusterloader2/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM golang:1.16.4 AS builder
```

Comment:
I'm really not excited about maintaining this as part of karpenter. Can we re-use someone else's build of this? Can we build it with `ko`?

Reply:
Removed images from this directory 👍